templates/web/common/includes/producers_platform_menu_links.tt.html

```diff
@@ -0,0 +1,10 @@
+<!-- start templates/[% template.name %] -->
+
+<li><label>[% lang("import_and_export_products") %]</label></li>
+<li><a href="/cgi/import_file_upload.pl">[% lang('import_product_data') %]</a></li>
+<li><a href="/cgi/import_photos_upload.pl">[% lang('import_product_photos') %]</a></li>
+<li><a href="/cgi/export_products.pl">[% lang('export_product_data_photos') %]</a></li>
+<li><a href="/cgi/remove_products.pl">[% lang('remove_products_from_producers_platform') %]</a></li>
+<li class="divider"></li>
```

Comment:
A much wanted feature by @manon-corneille is a link to the user guide. Could we add it here ? (but it should be translatable string as the link differs between french and others (english for now)).

Reply:
Good idea. I added a link to the user guide.